tests/host_reference/staggered_dslash_reference.h

```diff
@@ -23,7 +23,7 @@ void setDims(int *);
  * @param[in] dslash_type Dslash type
  */
 void stag_dslash(ColorSpinorField &out, const GaugeField &fat_link, const GaugeField &long_link,
```

Comment:
Needs updated doxygen

Reply:
This applies to multiple functions in this file

Reply:
196cef438